### DISCUSSION TITLE
Welcome to iOSDC2020-Talk-Sample Discussions!

### DISCUSSION THREAD

**Discussion by uhooi** [uhooi/iOSDC2020-Talk-Sample#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 123-ryu** [comment]
初めまして、宇田川ともうします。
昨晩GitHubをスマホから見つけ、aiを学習する上での問題をここを通して解決する事が出来ますか？お尋ねします。
